backend/Entities/Services/DbContext.cs

```diff
@@ -13,22 +13,14 @@ public class DbContext : IDbContext
 
         public DbContext()
         {
-            try
-            {
-                _myConnection.Open();
-            }
-            catch (Exception)
-            {
-                throw new InvalidOperationException();
-            }
+          
         }
 
         //Execute query, which return one string, where values separated by char
         public string ExecuteSqlQuery(string cmd, char separatedChar,Dictionary<string,object> param)
         {
             var result = new StringBuilder();
-            using (_myConnection)
-            {
+            _myConnection.Open();
```

Comment:
Because using 'using' force connection to dispose, thus we need either use Open() and Close() (as we do), or initialize new connection every time, passing it via aerguments (bad practice)

Reply:
If you in using block write "using(_myConnection.Open){}" you don`t need to close connection, because this block doing this